### ISSUE TITLE
Object.defineProperty behaves incorrectly in PhantomJS

### ISSUE THREAD

**Issue by Rich-Harris** [ariya/phantomjs#11856]
The following behaves correctly in browsers and node.js, but incorrectly in PhantomJS:

```js
var obj = {};

Object.defineProperty( obj, 'foo', { value: 'bar', configurable: true });
console.log( obj.foo );

Object.defineProperty( obj, 'foo', { value: 'baz', configurable: true });
console.log( obj.foo );
```

Correct result:

```
bar
baz
```

PhantomJS result:

```
bar
bar
```

**Comment by parasyte** [comment]
FWIW, this bug is fixed in PhantomJS 2.1.1

**Comment by Vitallium** [comment]
Thank! Closing this.
